store/store.go

```diff
@@ -395,6 +399,21 @@ func (s *Store) authenticateDevice(r *http.Request) {
 	}
 }
 
+func (s *Store) setStoreID(r *http.Request) {
+	storeID := s.fallbackStoreID
+	if s.authContext != nil {
+		cand, err := s.authContext.StoreID(storeID)
```

Comment:
yes, I tend to use cand quite a bit, it's storeIDCandidate but that's a bit long for a local variable for go